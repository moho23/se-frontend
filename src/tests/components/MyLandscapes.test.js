import React from "react";
import renderer from 'react-test-renderer'
import MyLandscapes from "../../MainProject/myLandscapes/myLandscapes.index"
import {mount, shallow} from "enzyme";

test('should render MyLandscapes correctly', () => {
    const wrapper = shallow(<MyLandscapes />);
    expect(wrapper).toMatchSnapshot();
  });

test("rendering MyLandscaes component <P> whithout crashing",() =>{
    const wrapper=shallow(<MyLandscapes/>);
    const p =(<p>!متاسفانه مکان ثبت شده ای نداری</p>);
    expect(wrapper.contains(p)).toEqual(true);
})

