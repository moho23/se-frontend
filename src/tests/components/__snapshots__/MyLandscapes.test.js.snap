// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MyLandscapes correctly 1`] = `ShallowWrapper {}`;
